lib/wpc_proto/internal_modules/proto_config.c

```diff
@@ -15,8 +15,194 @@
 #include "logger.h"
 
 
+/** Structure to hold config from node */
+typedef struct sink_config
+{
+    uint16_t stack_profile;
+    uint16_t hw_magic;
+    uint16_t ac_limit_min;
+    uint16_t ac_limit_max;
+    uint16_t app_config_max_size;
+    uint16_t version[4];
+    uint8_t max_mtu;
+    uint8_t ch_range_min;
+    uint8_t ch_range_max;
+    uint8_t pdu_buffer_size;
+
+    bool CipherKeySet;
+    bool AuthenticationKeySet;
+    uint8_t StackStatus;
+    uint16_t ac_range_min_cur;  // 0 means unset ?
+    uint16_t ac_range_max_cur;
+
+    app_role_t node_address;
+    app_role_t app_node_role;
+    wp_NodeRole wp_node_role;  // same as app_node_role, different format
+    net_addr_t network_address;
+    net_channel_t network_channel;
+
+    msap_app_config_data_write_req_pl_t app_config;
```

Comment:
Type is a bit strange.
I would have add app_config_data that is 80 bytes, sequence that is uint8_t (even if not used anymore nowadays) and diag interval, as a dedicated field

Reply:
as discussed, removed.